The processing instruction target matching "[xX][mM][lL]" is not allowed

Looks like I am seeing an error related to https://github.com/brianfrankcooper/YCSB/pull/758 ... any idea what is causing this?

Failed to auto configure default logger context
 Reported exception:
 ch.qos.logback.core.joran.spi.JoranException: Problem parsing XML document. See previously reported errors.
 at ch.qos.logback.core.joran.event.SaxEventRecorder.recordEvents(SaxEventRecorder.java:67)
 at ch.qos.logback.core.joran.GenericConfigurator.doConfigure(GenericConfigurator.java:134)
 at ch.qos.logback.core.joran.GenericConfigurator.doConfigure(GenericConfigurator.java:99)
 at ch.qos.logback.core.joran.GenericConfigurator.doConfigure(GenericConfigurator.java:49)
 at ch.qos.logback.classic.util.ContextInitializer.configureByResource(ContextInitializer.java:75)
 at ch.qos.logback.classic.util.ContextInitializer.autoConfig(ContextInitializer.java:150)
 at org.slf4j.impl.StaticLoggerBinder.init(StaticLoggerBinder.java:85)
 at org.slf4j.impl.StaticLoggerBinder.(StaticLoggerBinder.java:55)
 at org.slf4j.LoggerFactory.bind(LoggerFactory.java:128)
 at org.slf4j.LoggerFactory.performInitialization(LoggerFactory.java:108)
 at org.slf4j.LoggerFactory.getILoggerFactory(LoggerFactory.java:279)
 at org.slf4j.LoggerFactory.getLogger(LoggerFactory.java:252)
 at org.slf4j.LoggerFactory.getLogger(LoggerFactory.java:265)
 at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
 at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
 at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
 at java.lang.reflect.Method.invoke(Method.java:498)
 at com.allanbank.mongodb.util.log.Slf4jLogFactory.doGetLog(Slf4jLogFactory.java:103)
 at com.allanbank.mongodb.util.log.LogFactory.getLog(LogFactory.java:52)
 at com.allanbank.mongodb.Version.(Version.java:73)
 at com.allanbank.mongodb.builder.BatchedWrite.(BatchedWrite.java:94)
 at com.yahoo.ycsb.db.AsyncMongoDbClient.(AsyncMongoDbClient.java:105)
 at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
 at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
 at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
 at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
 at java.lang.Class.newInstance(Class.java:442)
 at com.yahoo.ycsb.DBFactory.newDB(DBFactory.java:40)
 at com.yahoo.ycsb.Client.main(Client.java:1058)
 Caused by: org.xml.sax.SAXParseException; lineNumber: 17; columnNumber: 6; The processing instruction target matching "[xX][mM][lL]" is not allowed.
 at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1239)
 at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:643)
 at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl.parse(SAXParserImpl.java:327)
 at ch.qos.logback.core.joran.event.SaxEventRecorder.recordEvents(SaxEventRecorder.java:61)
 ... 28 more
 09:02:08,385 |-INFO in ch.qos.logback.classic.LoggerContext[default] - Could NOT find resource [logback.groovy]
 09:02:08,385 |-INFO in ch.qos.logback.classic.LoggerContext[default] - Could NOT find resource [logback-test.xml]
 09:02:08,385 |-INFO in ch.qos.logback.classic.LoggerContext[default] - Found resource [logback.xml] at [jar:file:/home/tvoellm/ycsb-0.10.0/mongodb-binding/lib/mongodb-binding-0.10.0.jar!/logback.xml]
 09:02:08,394 |-INFO in ch.qos.logback.core.joran.spi.ConfigurationWatchList@ 6f75e72  - URL [jar:file:/home/tvoellm/ycsb-0.10.0/mongodb-binding/lib/mongodb-binding-0.10.0.jar!/logback.xml] is not of type file
 09:02:08,418 |-ERROR in ch.qos.logback.core.joran.event.SaxEventRecorder@ 69222c1  - XML_PARSING - Parsing fatal error on line 17 and column 6 org.xml.sax.SAXParseException; lineNumber: 17; columnNumber: 6; The processing instruction target matching "[xX][mM][lL]" is not allowed.
 at org.xml.sax.SAXParseException: The processing instruction target matching "[xX][mM][lL]" is not allowed.
 at at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.createSAXParseException(ErrorHandlerWrapper.java:203)
 at at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.fatalError(ErrorHandlerWrapper.java:177)
 at at com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:400)
 at at com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:327)
 at at com.sun.org.apache.xerces.internal.impl.XMLScanner.reportFatalError(XMLScanner.java:1438)
 at at com.sun.org.apache.xerces.internal.impl.XMLScanner.scanPIData(XMLScanner.java:723)
 at at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanPIData(XMLDocumentFragmentScannerImpl.java:1019)
 at at com.sun.org.apache.xerces.internal.impl.XMLScanner.scanPI(XMLScanner.java:691)
 at at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl$PrologDriver.next(XMLDocumentScannerImpl.java:912)
 at at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(XMLDocumentScannerImpl.java:606)
 at at com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.next(XMLNSDocumentScannerImpl.java:118)
 at at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:510)
 at at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:848)
 at at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:777)
 at at com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(XMLParser.java:141)
 at at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1213)
 at at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:643)
 at at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl.parse(SAXParserImpl.java:327)
 at at ch.qos.logback.core.joran.event.SaxEventRecorder.recordEvents(SaxEventRecorder.java:61)
 at at ch.qos.logback.core.joran.GenericConfigurator.doConfigure(GenericConfigurator.java:134)
 at at ch.qos.logback.core.joran.GenericConfigurator.doConfigure(GenericConfigurator.java:99)
 at at ch.qos.logback.core.joran.GenericConfigurator.doConfigure(GenericConfigurator.java:49)
 at at ch.qos.logback.classic.util.ContextInitializer.configureByResource(ContextInitializer.java:75)
 at at ch.qos.logback.classic.util.ContextInitializer.autoConfig(ContextInitializer.java:150)
 at at org.slf4j.impl.StaticLoggerBinder.init(StaticLoggerBinder.java:85)
 at at org.slf4j.impl.StaticLoggerBinder.(StaticLoggerBinder.java:55)
 at at org.slf4j.LoggerFactory.bind(LoggerFactory.java:128)
 at at org.slf4j.LoggerFactory.performInitialization(LoggerFactory.java:108)
 at at org.slf4j.LoggerFactory.getILoggerFactory(LoggerFactory.java:279)
 at at org.slf4j.LoggerFactory.getLogger(LoggerFactory.java:252)
 at at org.slf4j.LoggerFactory.getLogger(LoggerFactory.java:265)
 at at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
 at at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
 at at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
 at at java.lang.reflect.Method.invoke(Method.java:498)
 at at com.allanbank.mongodb.util.log.Slf4jLogFactory.doGetLog(Slf4jLogFactory.java:103)
 at at com.allanbank.mongodb.util.log.LogFactory.getLog(LogFactory.java:52)
 at at com.allanbank.mongodb.Version.(Version.java:73)
 at at com.allanbank.mongodb.builder.BatchedWrite.(BatchedWrite.java:94)
 at at com.yahoo.ycsb.db.AsyncMongoDbClient.(AsyncMongoDbClient.java:105)
 at at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
 at at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
 at at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
 at at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
 at at java.lang.Class.newInstance(Class.java:442)
 at at com.yahoo.ycsb.DBFactory.newDB(DBFactory.java:40)
 at at com.yahoo.ycsb.Client.main(Client.java:1058)

2016-07-08 09:02:08:468 0 sec: 0 operations; est completion in 0 seconds

--- a/mongodb/src/main/resources/logback.xml
+++ b/mongodb/src/main/resources/logback.xml
@@ -1,3 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
 <!-- 
 Copyright (c) 2016 YCSB contributors. All rights reserved.
 
@@ -14,7 +15,6 @@ implied. See the License for the specific language governing
 permissions and limitations under the License. See accompanying
 LICENSE file.
 -->
-<?xml version="1.0" encoding="UTF-8"?>
 <configuration>
 	<appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
 		<encoder>

